src/backend/src/lib.rs

```diff
@@ -147,12 +147,32 @@ fn set_config(arg: InitArg) {
     });
 }
 
+/// Runs housekeeping tasks immediately, then periodically:
+/// - `hourly_housekeeping_tasks`
+fn start_periodic_housekeeping_timers() {
+    // Run housekeeping tasks once, immediately but asynchronously.
+    let immediate = Duration::from_secs(0);
```

Comment:
There is a `Duration::ZERO` if interesting.

Reply:
Used.  Thanks!